source/drivers/driver-compatibility-reference.txt

```diff
@@ -645,56 +699,70 @@ MongoDB Compatibility
 .. list-table::
    :header-rows: 1
    :stub-columns: 1
-   :widths: 20 20 20 20 20
    :class: compatibility
 
    * - PHP Driver
      - MongoDB 2.4
      - MongoDB 2.6
      - MongoDB 3.0
      - MongoDB 3.2
+     - MongoDB 3.4
+
+   * - mongodb-1.2
```

Comment:
Sweet. I suspected as much but was having trouble digging through to figure out the canonical answer. Thanks!